lib/Option.cpp

```diff
@@ -355,6 +355,14 @@ static llvm::cl::opt<bool>
     debug_info("g", llvm::cl::init(false),
                llvm::cl::desc("Produce debug information."));
 
+static llvm::cl::opt<bool>
+    printf_support("enable-printf", llvm::cl::desc("Enable support for printf"),
```

Comment:
Is this an option we can have enabled by default?

Reply:
Probably in the future yes, but let's leave it off for the time being.